recipes/sdl/all/conanfile.py

```diff
@@ -77,6 +78,7 @@ class SDLConan(ConanFile):
         "opengles": True,
         "vulkan": True,
         "libunwind": True,
+        "hidapi": True,
```

Comment:
Do we need this `hidapi` option to be enabled by default. Adding new options by default may cause undesired behaviors on other users.
Which are the side effects of this option?
If your project needs it, wouldn't be safer to add it as a "False defaulted" option and enabled it on your side?

Thank you!

Reply:
it's on by default in SDL, so no change for existing users, see here: https://github.com/libsdl-org/SDL/blob/a5da9be5e64671c29595fe12132bbf105f71da05/CMakeLists.txt#L487C16-L487C22

Reply:
Okey, I see!
LGTM then